queries/cmake/highlights.scm

```diff
@@ -0,0 +1,101 @@
+[
+ (bracket_content)
+ (quoted_element)
+ (unquoted_argument)
+] @parameter
+
+(variable) @variable
+
+[
+ "$ENV"
+ "$CACHE"
+] @variable.builtin
+
+[
+ (normal_command)
+ (function_command)
+ (endfunction_command)
+ (macro_command)
+ (endmacro_command)
+ (message_command)
+] @function
+
+[
+ (if_command)
+ (elseif_command)
+ (else_command)
+ (endif_command)
+] @conditional
+
+[
+ (foreach_command)
+ (endforeach_command)
+ (while_command)
+ (endwhile_command)
+] @repeat
+
+[
+ "1"
+ "ON"
```

Comment:
wouldn't it be better to use constant highlighting for those? The vim regex highlights use bold text. You could also distinguish them in `AND` `NOT` operators, `FALSE` `TRUE` boolean

Reply:
Yes, I missed that when reading the contributing document. I will update them accordingly.